libvast/vast/format/mrt.hpp

```diff
@@ -286,6 +287,7 @@ struct attributes_parser : parser<attributes_parser<TypeCode14Policy>> {
       // can use the combinator style to chain the parser together, as opposed
       // to dispatching manually.
       auto t = f;
+      // clang-format off
```

Comment:
`clang-format` was disabled on line 268. Did you mean `// clang-format on`?

Reply:
Whoops, that comment was a bit overzealous. I've removed it.